lib/scheduler.js

```diff
@@ -221,7 +221,7 @@ function Scheduler(connection, options = {}) {
       }
       
       if (objectId.isValid(id)) {
-        event._id = id;
+        event._id = new mongo.ObjectID(id);
```

Comment:
Well seen !
If possible, use the same use as the line 244: `event._id = objectId(id)`

Reply:
fixed!